templates/zerver/help/import-users-and-channels-from-slack.md

```diff
@@ -0,0 +1,44 @@
+# Import users and channels from Slack
+
+{!follow-steps.md!} import users and channels from Slack to Zulip.
+
+1. Generate a Slack API token using Slack's [test token generator](https://api.slack.com/docs/oauth-test-tokens)
+   to import all of the necessary data.
+
+{!go-to-the.md!} [Your bots](/#settings/your-bots)
+{!settings.md!}
+
+3. Click on **Show/change your API key** button.
+
+4. Upon clicking the **Show/change your API key** button,
+   you will be asked to confirm your identity by entering
+   your password in the **Current password** field.
+
+5. Click the **Get API Key** button and copy the generated API Key.
+
+6. Fill in all settings in `api/integrations/slack/zulip_slack_config.py`:
+
+    * `SLACK_TOKEN` - token from point number 1.
+
+    * `ZULIP_USER` - e-mail of the user (the user that API key was generated for).
+
+    * `ZULIP_KEY` - API key from point number 4.
+
+    * `ZULIP_SITE` - Zulip API server URI.
+
+    !!! warn ""
+        **Note:** Please ensure the user has admin rights.
```

Comment:
I think we can make this clearer by saying "Please ensure that you have administrator rights before importing users and channels from Slack" at the top (after line 3)